doc/tutorial/alignment_file/alignment_file_solution2.cpp

```diff
@@ -76,35 +75,36 @@ int main()
 
     // read in reference information
     seqan3::sequence_file_input reference_file{tmp_dir/"reference.fasta"};
-    seqan3::concatenated_sequences<std::string> ref_ids{};
-    std::vector<std::vector<seqan3::dna5>> ref_seqs{};
+    std::vector<std::string> ref_ids{};
+    std::vector<seqan3::dna5_vector> ref_seqs{};
 
-    for (auto const & record : reference_file)
+    for (auto && record : reference_file)
     {
-        ref_ids.push_back(seqan3::get<seqan3::field::id>(record));
-        ref_seqs.push_back(seqan3::get<seqan3::field::seq>(record));
+        ref_ids.push_back(std::move(seqan3::get<seqan3::field::id>(record)));
+        ref_seqs.push_back(std::move(seqan3::get<seqan3::field::seq>(record)));
     }
 
-    seqan3::alignment_file_input mapping_file{tmp_dir/"mapping.sam",
-                                              ref_ids,
-                                              ref_seqs,
-                                              seqan3::fields<seqan3::field::id,
-                                                             seqan3::field::ref_id,
-                                                             seqan3::field::mapq,
-                                                             seqan3::field::alignment>{}};
+    auto const fields = seqan3::fields<seqan3::field::id,
```

Comment:
I would only declare a type here with using. Then construct the type inplace of the constructor argument.